### PR TITLE
Authentication Implementation

### PR DESCRIPTION
**Pull request by elisa242003**
Passport implementation (authentication), the token generated by JWT was used to validate registered users and subsequently obtain their information

### CODE REVIEW

File: src/auth/constants.ts
Comment (DiegoSHS) [inline src/auth/constants.ts:1]: *Security problem*
this shouldn't be plain text, use an enviroment variable instead

```js
export const jwtConstants = {
secret: env.JWT_SECRET
};
```

**Review by DiegoSHS — CHANGES_REQUESTED** [review]
JWT constants must be enviroment variables

File: src/auth/constants.ts
Comment (elisa242003) [reply src/auth/constants.ts:1]: Updated changes, in jwtConstants with an environment variable

Comment (elisa242003) [reply src/auth/constants.ts:1]: Updated changes, in jwtConstants with an environment variable

**Review by DiegoSHS — APPROVED** [review]

**Pull request merged**